publish-service/pom.xml

```diff
@@ -119,14 +124,14 @@
             <scope>compile</scope>
         </dependency>
         <dependency>
-            <groupId>com.jayway.restassured</groupId>
+            <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
             <scope>compile</scope>

```

Comment:
As i can see the version value is coming from remrem-parent which is 2.9.0 earlier, for groupId you have changed we dont have 2.9.0 version.. see here https://mvnrepository.com/artifact/io.rest-assured/rest-assured

Reply:
rest-assured uses release 5.3.0 now